src/main/protobuf/soc/playerState.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/inventory.proto";
+
+message PublicInventory {
+    required int32 numCards = 4;
+    required int32 numDevelopmentCards = 5;
+}
+
+message PerfectInventory {
+    required ResourceSet resourceCards = 1;
+    required DevelopmentCardSet playedDevCards = 2;
+    required DevelopmentCardSet canPlayDevCards = 3;
```

Comment:
Instead of dividing into `canPlayDevCards` and `playedDevCards`, why not tag each card with the turn it was purchased? This is public knowledge anyway?

Reply:
When a different player plays a development card it is not public info what turn they purchased the card.

Reply:
Really? I'm actually curious what (if anything) the rules say about this. Am I allowed to shuffle all my unplayed development cards, or am I supposed to leave them laid out before me (hence the turn is implicit)? Either is fine, but we have to make a decision.

Reply:
I can try to find this in the rules but I am pretty sure that turn is not implicit I do not think that is public info. Otherwise it is very easy to tell when you have a hidden VP and almost everyone who plays would be breaking the rules

Reply:
Not stated in the rules. I will assume it is not public information people can arrange their cards however they wish